### PR TITLE
Added Two new resources in free-courses-en.md

### PR DESCRIPTION
**Pull request by kraticodes**
## What does this PR do?
Added two new resources to free-courses-en.md ,one for Git and other for PHP.

### Description
Courses are open to all and are from geeksgod.com

### Why is this valuable (or not)?
These are great courses for beginners.

### How do we know it's really free?
Confirmed myself

### For book lists, is it a book? For course lists, is it a course? etc.
These are courses
## Checklist:
- [ :white_check_mark:] Read our [contributing guidelines]
- [ :white_check_mark:] Search for duplicates.
- [ :white_check_mark:] Include author(s) and platform where appropriate.
- [ :white_check_mark:] Put lists in alphabetical order, correct spacing.
- [ :white_check_mark:] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by kraticodes** [comment]
Hey ! Plz can you explain what's  the fault here?

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -319,6 +319,7 @@
 * [GitHowTo](https://githowto.com)
 * [How to Use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775) (Udacity)
 * [Introduction to Git and GitHub](https://www.coursera.org/learn/introduction-git-github) - Google (Coursera)
+* [Learn Version Control system with GIT](https://geeksgod.com/udemy-free-course/git-for-beginners/)
```
Comment (davorpa) [inline courses/free-courses-en.md:322]: Following guidelines:
- tracking sites / parameters are not accepted. The right link is https://www.eduonix.com/git-for-beginners and then, the title `GIT for beginners`
- should give creator attributions: `Maksym Rudnyi`
- should anotate platform: `Eduonix Learning Solutions`

So, try to fix it with:

```markdown
* [GIT for beginners](https://geeksgod.com/udemy-free-course/git-for-beginners/) - Maksym Rudnyi (Eduonix Learning Solutions)
```
And then, reorder to satisfy the list alphabetization rule.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -612,6 +613,7 @@
 * [PHP Programming Language Tutorial - Full Course](https://www.youtube.com/watch?v=OK_JCtrrv-c) - freeCodeCamp.org (2018)
 * [PHP Tutorial](https://www.tutorialrepublic.com/php-tutorial/) - TutorialRepublic
 * [PHP Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhwBToawjm9faF1ixePexft-) - Dani Krossing
+* [Learn Top Ten Frameworks In PHP By Building Projects](https://geeksgod.com/udemy-free-course/learn-top-ten-frameworks-in-php-by-building-projects/)
```
Comment (davorpa) [inline courses/free-courses-en.md:616]: Please:
- Reorder with the purpose of accomplish with the list alphabetization rule.
- Tracking sites / parameters are not accepted. Right link is https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects
- Should give platform / creator attributions: `Eduonix Learning Solutions`

```markdown
* [Learn Top Ten Frameworks In PHP By Building Projects](https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects) - Eduonix Learning Solutions
```

**Review by davorpa — CHANGES_REQUESTED** [review]
> Hey ! Plz can you explain what's the fault here?

@kraticodes, I give you deep feedback of this faults 

image
<details><summary>As linter report us (click to see details) :</summary>

```log
Run fpb-lint ./courses/
free-courses-en.md
      3:1-71:38  warning  Alphabetical ordering: swap l.4 and l.3      alphabetize-lists            remark-lint
       3:1-3:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       4:1-4:24  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       5:1-5:63  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       6:1-6:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       7:1-7:24  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       8:1-8:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
       9:1-9:31  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     10:1-10:26  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     11:1-11:10  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     12:1-12:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     13:1-13:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     14:1-14:26  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     15:1-15:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     16:1-16:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     17:1-17:32  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     18:1-20:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     19:3-19:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     20:3-20:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     21:1-21:34  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     22:1-22:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     23:1-23:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     24:1-24:40  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     25:1-25:14  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     26:1-26:12  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     27:1-27:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     28:1-28:27  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     29:1-29:14  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     30:1-30:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     31:1-40:21  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     32:3-32:28  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     33:3-33:19  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     34:3-34:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     35:3-35:23  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     36:3-36:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     37:3-37:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     38:3-38:34  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     39:3-39:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     40:3-40:21  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     41:1-41:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     42:1-42:28  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     43:1-43:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     44:1-44:14  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     45:1-45:40  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     46:1-46:24  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     47:1-47:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     48:1-48:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     49:1-49:15  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     50:1-50:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     51:1-51:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     52:1-52:14  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     53:1-53:80  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     54:1-54:28  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     55:1-57:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     56:3-56:22  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     57:3-57:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     58:1-58:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     59:1-59:10  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     60:1-60:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     61:1-61:16  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     62:1-62:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     63:1-63:24  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     64:1-65:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     65:3-65:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     66:1-66:20  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     67:1-69:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     68:3-68:24  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     69:3-69:18  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     70:1-70:66  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
     71:1-71:38  warning  Marker style should be `*`                   unordered-list-marker-style  remark-lint
  608:1-616:151  warning  Alphabetical ordering: swap l.616 and l.615  alphabetize-lists            remark-lint

⚠ 71 warnings
Error: Process completed with exit code 1.
```
</details>

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -316,10 +316,10 @@
 * [Git](https://www.youtube.com/playlist?list=PLFBirL3MAv29Vy_L7MmV2QaZLvAadFPHR) - Gwendolyn Faraday
 * [Git & GitHub Tutorial for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9goXbgTDQ0n_4TBzOO0ocPR) (The Net Ninja)
 * [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
+* [GIT for beginners](https://geeksgod.com/udemy-free-course/git-for-beginners/) - Maksym Rudnyi (Eduonix Learning Solutions)
```
Comment (davorpa) [inline courses/free-courses-en.md:319]: Following guidelines:
- tracking sites / parameters are not accepted. The right link is https://www.eduonix.com/git-for-beginners
- maybe complete it with note about need platform sign up

So, try to fix it with:

```markdown
* [GIT for beginners](https://www.eduonix.com/git-for-beginners) - Maksym Rudnyi (Eduonix Learning Solutions) *(account or email address required)*
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -607,13 +607,13 @@
 
 * [Laravel 5.8 Tutorial From Scratch](https://www.youtube.com/playlist?list=PLpzy7FIRqpGD0kxI48v8QEVVZd744Phi4) - Coder's Tape  (2019)
 * [Learn how to program: PHP](https://www.learnhowtoprogram.com/php) - Epicodus Inc.
+* [Learn Top Ten Frameworks In PHP By Building Projects](https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects) - Eduonix Learning Solutions
```
Comment (davorpa) [inline courses/free-courses-en.md:610]: Maybe complete it with note about need platform sign up

```markdown
* [Learn Top Ten Frameworks In PHP By Building Projects](https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects) - Eduonix Learning Solutions *(account or email address required)*
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Nice! List alphabetization was fixed :rocket:

- Now some minor changes.
- Remember also fix pending TOC suggested changes.

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 
+* [Index](#index)
 * [0 - MOOC](#0---mooc)
 * [Algorithms & Data Structures](#algorithms--data-structures)
 * [Android](#android)
 * [Assembly](#assembly)
 * [AutoIt](#autoit)
 * [Bash / Shell](#bash--shell)
 * [Bootstrap](#bootstrap)
-* [C](#c)
```
Comment (davorpa) [inline courses/free-courses-en.md:10]: Restore

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
 ### Index
 
+* [Index](#index)
```
Comment (davorpa) [inline courses/free-courses-en.md:3]: Remove

**Review by davorpa — CHANGES_REQUESTED** [review]
@kraticodes a last effort :wink:...

**Comment by kraticodes** [comment]
FINALLY! Thankyou @davorpa for guiding me all the way. 😄 🔥

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -1,13 +1,14 @@
 ### Index
 
-* [Index](#index)
+
```
Comment (davorpa) [inline courses/free-courses-en.md]: Maybe I should have said "remove line" in https://github.com/EbookFoundation/free-programming-books/pull/5578#discussion_r719655588 😶‍🌫️😣

If applied, it prevents conflicts with other PR #5581

**Review by davorpa — CHANGES_REQUESTED** [review]
> FINALLY! Thankyou @davorpa for guiding me all the way. smile fire

Yes, we get it. No lint errors. But.... last step... this is easy :grinning:

**Review by davorpa — APPROVED** [review]
@kraticodes, Forget last review (https://github.com/EbookFoundation/free-programming-books/pull/5578#pullrequestreview-768273086). I mistake due to not ui refresh with last changes.

Erick, ready to merge

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**